packages/charts/src/charts/line/gradient-filter.component.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from 'react';
+
+type Props = {
+  filterId: string;
+  firstOpacity: number;
+  lastOpacity: number;
+  color: string;
+};
+
+const GradientFilter: FC<Props> = ({
```

Comment:
I would consider using React `memo` here for better performance as all of properties are `primitive` types.